watchtower/wtwire/session_init_reply.go

```diff
@@ -0,0 +1,62 @@
+package wtwire
+
+import "io"
+
+// TODO(conner): enumerate fail codes
+
+// SessionInitReply is a message sent from watchtower to client in response to a
+// SessionInit message, and signals either an acceptance or rejection of the
+// proposed session parameters.
+type SessionInitReply struct {
+	// Code will be non-zero if the watchtower rejected the session init.
+	Code uint16
+
+	// RewardPkScript is the reward address to be included in any sweep
+	// transactions if the reward is not dusty.
+	RewardPkScript []byte
+
+	// TODO(conner): respond with payment request when full incentive model
+	// is implemented.
+}
+
+// A compile time check to ensure SessionInitReply implements the wtwire.Message
+// interface.
+var _ Message = (*SessionInitReply)(nil)
+
+// Decode deserializes a serialized SessionInitReply message stored in the passed
+// io.Reader observing the specified protocol version.
+//
+// This is part of the wtwire.Message interface.
+func (m *SessionInitReply) Decode(r io.Reader, pver uint32) error {
+	return ReadElements(r,
+		&m.Code,
+		&m.RewardPkScript,
+	)
+}
+
+// Encode serializes the target SessionInitReply into the passed io.Writer
+// observing the protocol version specified.
+//
+// This is part of the wtwire.Message interface.
+func (m *SessionInitReply) Encode(w io.Writer, pver uint32) error {
+	return WriteElements(w,
+		m.Code,
+		m.RewardPkScript,
+	)
+}
+
+// MsgType returns the integer uniquely identifying this message type on the
+// wire.
+//
+// This is part of the wtwire.Message interface.
+func (m *SessionInitReply) MsgType() MessageType {
+	return MsgSessionInitReply
+}
+
+// MaxPayloadLength returns the maximum allowed payload size for a SessionInitReply
+// complete message observing the specified protocol version.
+//
+// This is part of the wtwire.Message interface.
+func (m *SessionInitReply) MaxPayloadLength(uint32) uint32 {
+	return 100
```

Comment:
If we're also packaging a large-ish payment request (or other additional details depending on the final format of the e-cash tokens), then we may want to bump this up a bit.